test/cache.c

```diff
@@ -4145,6 +4145,16 @@ check_flush_cache__flush_ops(H5F_t *file_ptr)
     struct fo_flush_entry_check     *checks        = NULL;
     H5C_t                           *cache_ptr     = file_ptr->shared->cache;
 
+    /* Per-test variables. Each test sets these. */
+    int          test_num;
+    unsigned int flush_flags;
+    int          spec_size;
+    int          check_size;
+    unsigned     init_expected_index_len;
+    size_t       init_expected_index_size;
+    unsigned     expected_index_len;
+    size_t       expected_index_size;
```

Comment:
There are so many tests w/ repeated block variables in this function, it generates a stack size warning w/ XCode's clang